android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/NoOpAnnotationPublisher.kt

```diff
@@ -0,0 +1,9 @@
+package com.stadiamaps.ferrostar.core.annotation
+
+import com.stadiamaps.ferrostar.core.NavigationState
+
+class NoOpAnnotationPublisher : AnnotationPublisher<Any> {
```

Comment:
Low: Could we specialize this to `Unit` so that it's a dead end instead of someone accidentally messing with `Any` which we all know is nothing.

Reply:
done.